src/js/wrapper.ts

```diff
@@ -39,5 +41,90 @@ export const NATIVE = {
     return RNSentry.sendEvent(event);
   },
 
+  /**
+   * Starts native with dsn string and options.
+   * @param dsn string
+   * @param options ReactNativeOptions
+   */
+  async startWithDsnString(
+    dsn: string,
+    options: ReactNativeOptions
+  ): Promise<boolean> {
+    if (this.isNativeClientAvailable()) {
+      // tslint:disable-next-line: no-unsafe-any
+      return RNSentry.startWithDsnString(dsn, options);
+    }
+    return Promise.reject();
+  },
+
+  /**
+   * Fetches the release from native
+   */
+  async fetchRelease(): Promise<{
+    build: string;
+    id: string;
+    version: string;
+  }> {
+    if (this.isNativeClientAvailable()) {
+      // tslint:disable-next-line: no-unsafe-any
+      return RNSentry.fetchRelease();
+    }
+    return Promise.reject();
+  },
+
+  /**
+   * Fetches the device contexts. Not used on Android.
+   */
+  async deviceContexts(): Promise<object> {
+    if (this.isNativeClientAvailable()) {
+      // tslint:disable-next-line: no-unsafe-any
+      return RNSentry.deviceContexts();
+    }
+    return Promise.reject();
+  },
+
+  /**
+   * Sets log level in native
+   * @param level number
+   */
+  setLogLevel(level: number): void {
+    // tslint:disable-next-line: no-unsafe-any
+    RNSentry.setLogLevel(level);
```

Comment:
We should check if RNSentry is available before all calls to it.

Maybe a gatekeeper `if` at the start of every function.
We can have a cached Promise that we resolve/reject once and rely on that then.